### DISCUSSION TITLE
What's the sound of 24 DX7's?

### DISCUSSION THREAD

**Discussion by DutchBigEd** [probonopd/MiniDexed#540] [Show and tell]
https://youtu.be/XHixhaHuecc

Thanks to Banana71 for the midi-file and creating the sounds (except for the drums, I used an YMF704 for it).

**Comment by probonopd** [comment]
Wow! What's better than a MiniDexed? Three MiniDexeds! :1st_place_medal:

**Comment by diyelectromusic** [comment]
If you can drop in some Pi 4's I could send you my 16-TG mod... :)

**Reply by DutchBigEd** [reply]
Sorry, can't...

**Comment by Banana71** [comment]
Hi Ed,

great gear.
You didn't assign the tracks quite correctly. The solo also plays on midi channel 1 and the bass on channel 3.
But so cool.

**Reply by DutchBigEd** [reply]
First I have each minidexed play solo a few measures of It's instrument, after that all the minidexed play together.

**Reply by Banana71** [reply]
I checked your video again, the white device in the top right should be set to performance 140.

**Reply by DutchBigEd** [reply]
I have a Cube Mi5, which has 5 harware midi outputs.
To the 1st output I assigned Minidexed 1, playing Synth, 2nd output plays Synth Solo on Minidexed 2, 3rd output goes to Minidexed 3 (black) on channel 3 and plays Bass. The 5th output goes to the YMF704 module and playes the Timbale on channel 1 and drums on channel 10.
4th output isn't used at this moment.
![20230817_200605](https://github.com/probonopd/MiniDexed/assets/17274454/5a90bbb1-9b89-4f9a-be2c-1f98081c2532)

**Reply by soyersoyer** [reply]
Which performances are these?

**Comment by Banana71** [comment]
https://github.com/Banana71/Soundplantage

There are some performances in the folder 003_Project_Perf . Please note that the MIDI channels have been adapted to the projects.
Enjoy the Silence
Sounds like a Melodie
Sweet Dreams
